README.md

```diff
@@ -56,6 +56,22 @@ make serve
 
 This will start the local Hugo server on port 1313. Open up your browser to http://localhost:1313 to view the website. As you make changes to the source files, Hugo updates the website and forces a browser refresh.
 
+## Rebuilding the API reference pages
+
+The API reference pages located in `content/en/docs/reference/kubernetes-api/ref` are built from the Swagger specifications, using https://github.com/feloy/kubernetes-resources-reference.
+
+If you want to rebuild these pages, you first have to pull the `kubernetes-resources-reference` submodule:
```

Comment:
nit: Will this be optional or required? I think this sub-module is required, similar to the theme.

Reply:
You may not need to update this README. Instead update the project's README.